From 858412eb5617b30ae1cecd6d520f5ec62bf8af8d Mon Sep 17 00:00:00 2001
From: Paul Seidler <pl.seidler@gmail.com>
Date: Sun, 24 Jul 2011 15:14:40 +0000
Subject: [PATCH] don't use bundled libebml and libmatroska
Upstream: no, upstream decided to use bundled lib versions as they
          haven't enough manpower to check for compatibilities
           
---
 configure.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.in b/configure.in
index c6c64f6..4e352ed 100644
--- a/configure.in
+++ b/configure.in
@@ -29,9 +29,9 @@ m4_include(ac/nl_langinfo.m4)
 m4_include(ac/ogg.m4)
 m4_include(ac/vorbis.m4)
 m4_include(ac/flac.m4)
-dnl m4_include(ac/ebml.m4)
-dnl m4_include(ac/matroska.m4)
-m4_include(ac/ebml_matroska_internal.m4)
+m4_include(ac/ebml.m4)
+m4_include(ac/matroska.m4)
+dnl m4_include(ac/ebml_matroska_internal.m4)
 m4_include(ac/expat.m4)
 m4_include(ac/zlib.m4)
 m4_include(ac/lzo.m4)
-- 
1.7.6

